### ISSUE TITLE
charge_immediately parameter is unused

### ISSUE THREAD

**Issue by streeter** [pinax/pinax-stripe#249]
The [`charge_immediately` parameter to `actions.subscriptions.update()`](https://github.com/pinax/pinax-stripe/blob/cd44858cefbe9992e657da276149ad51eb4d20b8/pinax/stripe/actions/subscriptions.py#L172-L182) is unused.

I'm assuming this means the user is always immediately charged.

**Comment by streeter** [comment]
I'm thinking that the code doesn't charge the user immediately, as here's the [old code location](https://github.com/pinax/pinax-stripe/blob/2.0.0/payments/models.py#L573).

**Comment by ossanna16** [comment]
Thanks for reporting this @streeter!

**Comment by paltman** [comment]
You are right.  `prorate` takes care of the concept of charging immediately.

**Comment by paltman** [comment]
Actually, you are wrong. :)  At least at this point in time.  It is [definitely being used](https://github.com/pinax/pinax-stripe/blob/master/pinax/stripe/actions/subscriptions.py#L195).

**Comment by streeter** [comment]
@paltman, I can't tell if your latest comment was sarcastic or not, so I'll assume it was meant as a joke. I linked to the code revision where it wasn't being used in the issue description (https://github.com/pinax/pinax-stripe/blob/cd44858cefbe9992e657da276149ad51eb4d20b8/pinax/stripe/actions/subscriptions.py#L172-L182), so it was definitely not used at the time I created this issue.

Glad to see that it was fixed since then.
